src/commands/app/pack.js

```diff
@@ -252,28 +252,49 @@ class Pack extends BaseCommand {
    * @param {object} appConfig the app's configuration file
    */
   async addCodeDownloadAnnotation (appConfig) {
-    // get the configFiles that have runtime manifests
-    const configFiles = []
-    for (const [, value] of Object.entries(appConfig.includeIndex)) {
-      const { key } = value
-      if (key === 'runtimeManifest' || key === 'application.runtimeManifest') {
-        configFiles.push(value)
+    // get each annotation key relative to the file it is defined in
+    const fileToAnnotationKey = {}
+    Object.entries(appConfig.all).forEach(([ext, extConf]) => {
+      if (extConf.manifest?.full?.packages) {
```

Comment:
wdyt about a `.filter` before the forEach instead of the `if` inside it? Minimal perf hit but I think its more readable vs all the nesting
```
Object.entries(appConfig.all)
    .filter(([ext, extConf]) => extConf.manifest?.full?.packages)
    .forEach(([ext, extConf]) => {
```

Reply:
sure thanks